DynamicScreen.c

```diff
@@ -0,0 +1,61 @@
+/*
+htop - DynamicScreen.c
+(C) 2022 Sohaib Mohammed
+(C) 2022 htop dev team
+(C) 2022 Red Hat, Inc.  All Rights Reserved.
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h" // IWYU pragma: keep
+
+#include "DynamicScreen.h"
+
+#include <stdbool.h>
+#include <stddef.h>
+
+#include "Hashtable.h"
+#include "Platform.h"
+#include "XUtils.h"
+#include "config.h"
+
+
+Hashtable* DynamicScreens_new(Settings* settings) {
+   return Platform_dynamicScreens(settings);
+}
+
+void DynamicScreens_delete(Hashtable* dynamics) {
+   if (dynamics) {
+      Platform_dynamicScreensDone(dynamics);
+      Hashtable_delete(dynamics);
+   }
+}
+
+typedef struct {
+   unsigned int key;
```

Comment:
Shouldn't this be `ht_key_t`, as this is effectively directly passed to the `Hashtable` implementation?